tests/qt/test_file_path_options.py

```diff
@@ -0,0 +1,119 @@
+import os
+import pathlib
+from unittest.mock import patch
+
+import pytest
+from PySide6.QtGui import (
+    QAction,
+)
+from PySide6.QtWidgets import QMenu, QMenuBar
+
+from tagstudio.core.enums import SettingItems
+from tagstudio.core.library.alchemy.library import LibraryStatus
+from tagstudio.qt.modals.settings_panel import SettingsPanel
+from tagstudio.qt.widgets.preview_panel import PreviewPanel
+
+
+# Tests to see if the file path setting is applied correctly
+@pytest.mark.parametrize(
+    "filepath_option", ["show full path", "show relative path", "show only file name"]
+)
+def test_filepath_setting(qtbot, qt_driver, filepath_option):
+    settings_panel = SettingsPanel(qt_driver)
+    qtbot.addWidget(settings_panel)
+
+    # Mock the update_recent_lib_menu method
+    with patch.object(qt_driver, "update_recent_lib_menu", return_value=None):
+        # Set the file path option
+        settings_panel.filepath_combobox.setCurrentText(filepath_option)
```

Comment:
*(In the context of enums)* And just like the combobox in the settings menu we'd be using setting from indexes rather than setting the text directly here now
```suggestion
        settings_panel.filepath_combobox.setCurrentIndex(filepath_option)
```